lib/convertkit/form.rb

```diff
@@ -43,7 +44,7 @@ class Client
     def forms()
       raw   = get_request("/forms")
       forms = []
-      return forms unless raw["forms"].present?
+      return forms unless raw["forms"].any?
```

Comment:
what is the sense of this and below change?

Reply:
nil.any? => NoMethodError: undefined method `any?' for nil:NilClass
nil.present? => false

Reply:
well when you request a list of tags/forms the key `tags/forms` returns always an array, the worse case returns an empty array, but yeah probably is safer to use `present?`

Reply:
Actually I was playing with it and `.present?` is a rails method not a ruby one....so we can't use it here (I mean we could but I need to add `active_support`) solution is `.nil?`:
```
[1].nil? -> false
[].nil? -> false but then [].each will just return []
nil.nil? -> true
```